### DISCUSSION TITLE
WalletConnect v2 not working in production wagmi@0.12.12

### DISCUSSION THREAD

**Discussion by devsalman247** [wevm/wagmi#2252] [Q&A]
We're trying to use multiple connectors in our wagmi config so that users can connect their wallet of choice.... We are not using web3modal...WalletConnectConnector v2 is working fine in local environment but giving an error on production on console (see ss) and modal is not opening...
![walletconnectconnector v2](https://user-images.githubusercontent.com/112937676/235634556-d05bb9f8-dd28-4d18-bc06-40870a04e48c.png)
Here is our wagmi config :
`import { createClient, configureChains, mainnet } from "wagmi";
import { publicProvider } from "wagmi/providers/public";
import { MetaMaskConnector } from "wagmi/connectors/metaMask";
import { CoinbaseWalletConnector } from "wagmi/connectors/coinbaseWallet";
import { WalletConnectConnector } from "wagmi/connectors/walletConnect";
import { bsc } from "wagmi/chains";

export const walletConnectProjectId = "ba7804e457fbb5f1375cbdc14e679617";

const { chains, provider, webSocketProvider } = configureChains([mainnet, bsc], [publicProvider()]);

export const client = createClient({
	autoConnect: true,
	connectors: [
		new MetaMaskConnector({ chains }),
		new CoinbaseWalletConnector({
			chains,
			options: {
				appName: "NFT Promo",
			},
		}),
		new WalletConnectConnector({
			chains,
			options: {
				projectId: walletConnectProjectId,
			},
		}),
	],
	provider,
	webSocketProvider,
});`

**Comment by victorpavlenko** [comment]
Did you manage to figure it out? 
For me the problem was solved with setting the correct hosts for your projectId for wallet
